prov/rxm/src/rxm_ep.c

```diff
@@ -849,7 +849,7 @@ rxm_ep_send_common(struct fid_ep *ep_fid, const struct iovec *iov, void **desc,
 		      tx_buf->hdr.desc, 0, tx_entry);
 	if (OFI_UNLIKELY(ret)) {
 		if (ret == -FI_EAGAIN)
-			rxm_cq_progress(rxm_ep);
+			rxm_ep_progress(&rxm_ep->util_ep);
```

Comment:
The `rxm_ep_progress()` function calls the `rxm_cq_progress()` function that is defined in the `rxm_cq.c` as static. It can't be called here directly.

Reply:
This is the proposed code flow:
```
rxm_ep_progress(&rxm_ep->util_ep);

void rxm_ep_progress(struct util_ep *util_ep)
{
    struct rxm_ep *rxm_ep = container_of(util_ep, struct rxm_ep, util_ep);
    rxm_cq_progress(rxm_ep);
}
```
Honestly, I would remove the rxm_ep_progress() call.  Rename rxm_cq_progress() -> rxm_ep_progress() -- because it takes the ep as a parameter rather than the cq.  And consider moving the call into a header file.  All of this is dependent on how the progress/progress_ex calls are restructured.

Reply:
Nice point. Done